igvm/hypervisor.py

```diff
@@ -215,6 +190,40 @@ def check_vm(self, vm, offline):
                     .format(*hw_pair)
                 )
 
+        # HV in supported state?
+        if self.dataset_obj['state'] not in ['online', 'online_reserved']:
+            raise InvalidStateError(
+                'Hypervisor "{}" is not in online state ({}).'
+                .format(self.fqdn, self.dataset_obj['state'])
+            )
+
+        # Enough CPUs?
+        if vm.dataset_obj['num_cpu'] > self.dataset_obj['num_cpu']:
+            raise HypervisorError(
+                'Not enough CPUs. Destination Hypervisor has {0}, '
+                'but VM requires {1}.'
+                .format(self.dataset_obj['num_cpu'], vm.dataset_obj['num_cpu'])
+            )
+
+        # Proper VLAN?
+        if not self.get_vlan_network(vm.dataset_obj['intern_ip']):
```

Comment:
This check also has taken place already see: 

https://github.com/innogames/igvm/blob/fdc85aa8825a07abf24448df0b7081f817d91f80/igvm/commands.py#L833

Reply:
Same as with state, we check a HV for all parameters while its locked. However Serveradmin users could change this without locking.

If we don't care for that, then this is redundant code, yes.

Reply:
lets wait for feedback from @TuxPowered42 but addresse this in another pr since the logic is the same as before, just in another place

Reply:
If it's redundant we could make a separate PR to fix it. I would prefer not to change it too much with other changes @Woellchen did.